events/models.py

```diff
@@ -21,10 +20,10 @@ class Event(models.Model):
 
     name = models.CharField(max_length=100)
 
-    datetime = models.DateField()
+    datetime = models.DateTimeField(default=datetime.date.today())
```

Comment:
@ImaneChafi can you check this? I have no clue what is going on here

Reply:
I thought this was already changed

Reply:
Are we only dealing with dates and not specific times for events?

Reply:
I think we are dealing with both dates and times but I had the impression this was changed, Imane and I spent hours on this so I want to check

Reply:
Sorry, I may have wrote this or kept my original code while merging

Reply:
No, I actually just checked the source and the somehow it got changed from date back to datetime, but I am confused why which is why I want @ImaneChafi input before doing anything

Reply:
On a side note, I thought gitignore took care of the *.pyc files

Reply:
I thought I added it in there but it apparently didn't cover everything. In the future, just do a quick `git status` to make sure you are not adding any of the cache files before committing

Reply:
Yeah it was changed to models.DateField() because the original one wasn't working on migrations. I'll try to find if its making migrations errors

Reply:
It's fine so if you leave it like this it's going to ask you an input for null values on making migrations, and then you just need to add info in the prompt like you would normally do, I also ran the application and it's working fine. :    
`$ python manage.py makemigrations
System check identified some issues:

WARNINGS:
events.Event.date: (fields.W161) Fixed default value provided.
        HINT: It seems you set a fixed date / time / datetime value as default for this field. This may not be what you want. If you want to have the current date as default, use `django.utils.timezone.now`
You are trying to add a non-nullable field 'author' to comment without a default; we can't do that (the database needs something to populate existing rows).
Please select a fix:
 1) Provide a one-off default now (will be set on all existing rows with a null value for this column)
 2) Quit, and let me add a default in models.py
Select an option: 1
Please enter the default value now, as valid Python
The datetime and django.utils.timezone modules are available, so you can do e.g. timezone.now
Type 'exit' to exit this prompt
>>> exit`

However, the app is failing on migrations and it's giving me this error : 

`Running migrations:
  Applying events.0002_auto_20200220_2216...Traceback (most recent call last):
  File "C:\Users\user\Miniconda3\lib\site-packages\django\db\backends\utils.py", line 86, in _execute
    return self.cursor.execute(sql, params)
psycopg2.errors.StringDataRightTruncation: value too long for type character varying(20)

`